pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -58,6 +46,13 @@ func NewResourceSyncController(
 	); err != nil {
 		return nil, err
 	}
+	// this ca bundle contains certs provided by the kube-apiserver to verify aggregator client certs
+	if err := resourceSyncController.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "kube-apiserver-aggregator-client-ca"},
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "aggregator-client-ca"},
```

Comment:
this is where our operator places the unified ca bundle

Reply:
Who creates this? I see `operator-ns/managed-aggregator-client-ca`.

Reply:
> Who creates this? I see `operator-ns/managed-aggregator-client-ca`.

happens in the targetconfigcontroller https://github.com/openshift/cluster-kube-apiserver-operator/pull/216/files#diff-6501874255487b3161b3dfc0f3f34070R239